client/components/jetpack/card/jetpack-product-card/display-price/paid.tsx

```diff
@@ -31,7 +32,9 @@ const Placeholder: React.FC< OwnProps > = ( { billingTerm, expiryDate, discounte
 				rawPrice={ 0.01 }
 				currencyCode="USD"
 			/>
-			{ discountedPrice && <PlanPrice discounted rawPrice={ 0.01 } currencyCode="USD" /> }
```

Comment:
Just a couple of questions here:

Is `discounted` a boolean flag?
Any idea why `rawPrice` is set to 0.01? Is it some sort of a pre-existing hack?

Reply:
> Is discounted a boolean flag?
Any idea why rawPrice is set to 0.01? Is it some sort of a pre-existing hack?

I'm not sure, I haven't touched those. The only thing I changed here is that its not `discountedPrice &&` but `isNumber( discountedPrice ) &&` for the logic check.

That's needed because discountedPrice would be 0 for Advanced plan now, which is falsy, so we rather have to check if it's a number to have the wanted behaviour